### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by dubeyanurag**
This PR adds pagination and swipe refresh layout to issues.

### CODE REVIEW

File: app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java [hunk 1]
```diff
@@ -164,11 +193,16 @@ private void setupRecyclerview() {
     }
 
     private void fetchIssues() {
-        progressBar.setVisibility(View.VISIBLE);
-        adapter.clearData();
-
+        loading=true;
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java]: spaces next to `=`

File: app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java [hunk 2]
```diff
@@ -164,11 +193,16 @@ private void setupRecyclerview() {
...
+        swipeRefreshLayout.setEnabled(false);
+        if(page==1) {
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java]: spaces next to `==`

File: app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java [hunk 3]
```diff
@@ -164,11 +193,16 @@ private void setupRecyclerview() {
...
+            progressBar.setVisibility(View.VISIBLE);
+            adapter.clearData();
+        }
+        else
+            swipeRefreshLayout.setRefreshing(true);
         String language = Utils.getLanguagePreference(getActivity());
-        GithubRepository.getInstance().findIssues(language)
+        GithubRepository.getInstance().findIssues(language,page)
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java]: space after `,`

File: app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java [hunk 4]
```diff
@@ -180,6 +214,9 @@ public void onCompleted() {
                     @Override
                     public void onError(Throwable e) {
                         progressBar.setVisibility(View.GONE);
+                        swipeRefreshLayout.setRefreshing(false);
+                        swipeRefreshLayout.setEnabled(true);
+                        loading=false;
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java]: spaces between `=`

File: app/src/main/java/com/naman14/hacktoberfest/network/api/services/SearchService.java [hunk 5]
```diff
@@ -17,7 +17,7 @@
     Observable<SearchResponse> findPRs(@Query(encoded = true, value = "q") String searchQuery);
 
     @GET(ApiEndPoints.SEARCH + "/issues")
-    Observable<SearchResponse> findIssues(@Query(encoded = true, value = "q") String searchQuery);
+    Observable<SearchResponse> findIssues(@Query(encoded = true, value = "q") String searchQuery,@Query(encoded = true, value = "page") int page);
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/network/api/services/SearchService.java]: space after `,`

File: app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java [hunk 6]
```diff
@@ -188,8 +225,13 @@ public void onError(Throwable e) {
                     @Override
                     public void onNext(List<Issue> response) {
                         progressBar.setVisibility(View.GONE);
-                        adapter.setData(response);
-
+                        swipeRefreshLayout.setRefreshing(false);
+                        swipeRefreshLayout.setEnabled(true);
+                        loading=false;
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java]: spaces between `=`

File: app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java [hunk 7]
```diff
@@ -188,8 +225,13 @@ public void onError(Throwable e) {
...
+                        if(page==1)
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java]: spaces between `==`

File: app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java [hunk 8]
```diff
@@ -253,8 +295,8 @@ private void hide() {
 
     private void saveAndhide() {
         if (confirmSaveContainer.getVisibility() == View.VISIBLE) {
+            page=1;
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java]: spaces between `=`

File: app/src/main/java/com/naman14/hacktoberfest/network/repository/GithubRepository.java [hunk 9]
```diff
@@ -47,8 +47,8 @@ public static GithubRepository getInstance() {
     }
 
 
-    public Observable<List<Issue>> findIssues(String language) {
-        return apiManager.getSearchApi().findIssues(Utils.getHacktoberfestIssuesQuery(language))
+    public Observable<List<Issue>> findIssues(String language,int page) {
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/network/repository/GithubRepository.java]: space after `,`

File: app/src/main/java/com/naman14/hacktoberfest/network/repository/GithubRepository.java [hunk 10]
```diff
@@ -47,8 +47,8 @@ public static GithubRepository getInstance() {
...
+        return apiManager.getSearchApi().findIssues(Utils.getHacktoberfestIssuesQuery(language),page)
```
Comment (Empty2k12) [inline app/src/main/java/com/naman14/hacktoberfest/network/repository/GithubRepository.java]: space after `,`

**Comment by dubeyanurag** [comment]
Okay, will make a commit with fixed spaces. Meanwhile you can review the functionality.

**Pull request merged**